### PR TITLE
MediaImpacts first testintegration

### PR DESCRIPTION
**Pull request by renebaudisch**
changes in docfiles stashed, why do we get 3p unknown on localhost?

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const adPrefetch = {
   doubleclick: 'https://www.googletagservices.com/tag/js/gpt.js',
   a9: 'https://c.amazon-adsystem.com/aax2/assoc.js',
   adsense: 'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js',
+  mediaimpact: 'https://ec-ns.sascdn.com/diff/251/divscripte/t.js'
```
Comment (cramforce) [inline ads/_config.js]: Is this URL stable and not expected to change?

File: ads/mediaimpact.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import {writeScript} from '../src/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+
+export function mediaimpact(global, data) {
+  const src = "https://ec-ns.sascdn.com/diff/251/bild.js";
```
Comment (cramforce) [inline ads/mediaimpact.js]: Should this go into prefetch as well?

File: ads/mediaimpact.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  writeScript(global, src);
+  /*global.SmartAdServerAjax = function() {
```
Comment (cramforce) [inline ads/mediaimpact.js]: Is this related to https://github.com/ampproject/amphtml/pull/1808

File: ads/_config.js (hunk 1 above)
Comment (renebaudisch) [inline ads/_config.js]: At the moment it's for testing only, will be static to something like "https://ec-ns.sascdn.com/diff/251/amp/mediaimpact.js" when finished. Till then t.js does have everything we need for testing

File: ads/mediaimpact.js (hunk 2 above)
Comment (renebaudisch) [inline ads/mediaimpact.js]: We do have "https://ec-ns.sascdn.com/" in preconnect, if it's better to set it also to prefetch i would add the server root to it.

File: ads/mediaimpact.js (hunk 3 above)
Comment (renebaudisch) [inline ads/mediaimpact.js]: Kind of. We, MediaImpact, use SmartAdServer for delivery but we have our own lib to generate the adcalls. For historical reasons we have to use the same function-calls but treat them separatly to SmartAdServer in general.

So it's related but completly different.

Comment (cramforce) [inline ads/mediaimpact.js]: OK. Trying to understand whether this is a new ad server or just configuration of another one.

Comment (renebaudisch) [inline ads/mediaimpact.js]: Configuration for SmartAdServer done by MediaImpact :-)

Comment (cramforce) [inline ads/mediaimpact.js]: Is this an option for your use case

https://github.com/ampproject/amphtml/blob/master/builtins/amp-ad.md#running-ads-from-a-custom-domain

You can intercept the ad call and change configuration

**Comment by renebaudisch** [comment]
we're looking forward to distribute the cla, but we also changeds code, so i had to close this request

**Pull request closed without merging**